web/apps/labelstudio/src/pages/Projects/ProjectsList.jsx

```diff
@@ -39,12 +39,12 @@ export const EmptyProjectsList = ({ openModal }) => {
     <Block name="empty-projects-page">
       <Elem name="heidi" tag="img" src={absoluteURL("/static/images/opossum_looking.png")} />
       <Elem name="header" tag="h1">
-        Heidi doesn’t see any projects here!
+        Heidi doesn't see any projects here!
```

Comment:
I don't think we need to change nice apostrophe in a regular text
```suggestion
        Heidi doesn’t see any projects here!
```